docker-micro-benchmark/helpers/docker_helpers.go

```diff
@@ -17,40 +17,75 @@ limitations under the License.
 package helpers
 
 import (
+	"encoding/json"
 	"fmt"
+	"io"
 	"math/rand"
 	"strconv"
 	"sync"
 	"time"
 
-	docker "github.com/fsouza/go-dockerclient"
+	docker "github.com/docker/engine-api/client"
+	"github.com/docker/engine-api/types"
+	"github.com/docker/engine-api/types/container"
+	"github.com/docker/engine-api/types/events"
+	"github.com/docker/engine-api/types/network"
+	"github.com/docker/engine-api/types/strslice"
 	"github.com/juju/ratelimit"
+	"golang.org/x/net/context"
+)
+
+const (
+	TestImage   = "ubuntu"
+	TestCommand = "ubuntu"
```

Comment:
I've changed it before. However I met some problem with Godep, and maybe mass up the version a little during trying to fix the Godep. :)